src/main/java/io/anserini/index/IndexCollection.java

```diff
@@ -78,6 +79,9 @@
     @Option(name = "-keepStopwords", usage = "boolean switch to keep stopwords")
     public boolean keepStopwords = false;
 
+    @Option(name = "-keepRetweets", usage = "Boolean switch to keep retweets while indexing")
```

Comment:
And here we have an interesting debate, consistency vs. correctness. Consistency would say to lowercase `Boolean` in the help message, correctness would say to change all the others. In this case because it's likely lowercased in other files across the project, consistency probably wins.